plugins/processors/dcos_metadata/dcos_metadata.go

```diff
@@ -297,15 +308,16 @@ func mapExecutorNames(ge *agent.Response_GetExecutors) map[string]string {
 }
 
 // mapTaskLabels returns a map of all task labels prefixed DCOS_METRICS_
-func mapTaskLabels(labels *mesos.Labels) map[string]string {
+// or included in list of whitelisted labels
+func mapTaskLabels(labels *mesos.Labels, whitelisted map[string]bool) map[string]string {
 	results := map[string]string{}
 	if labels != nil {
 		for _, l := range labels.GetLabels() {
 			k := l.GetKey()
-			if len(k) > 13 {
-				if k[:13] == "DCOS_METRICS_" {
-					results[k[13:]] = l.GetValue()
-				}
+			if strings.HasPrefix(k, "DCOS_METRICS_") {
```

Comment:
What do you think about making this prefix configurable too?

Reply:
I like the idea - `whitelist_prefix` or something like that?